Project/src/MakeCall/MakeCall.js

```diff
@@ -1040,9 +1052,10 @@ this.deviceManager.on('selectedSpeakerChanged', () => { console.log(this.deviceM
                             </div>
                         }
                         {
-                            this.state.call && !this.state.callSurvey && !this.state.isPreCallDiagnosticsCallInProgress &&
+                            this.state.call && this.state.callJoinType && !this.state.callSurvey && !this.state.isPreCallDiagnosticsCallInProgress &&
                             <CallCard

```

Comment:
This CallCard component still gets used for incoming call.

Reply:
The callJoinType is also needed for incoming all types as well.

Reply:
It is not clear what we are trying to achieve with introducing callJoinType. Let you, Cris and I, meet up and discuss about this.

Reply:
We do not have any signal in CallCard to distinguish between a teams call & ACS call. The value that gets assigned tells us whether it is a teams call, rooms call or acs call. It is important to know that because we would apply Reaction API call based on call type.